.github/workflows/automated-tests.yml

```diff
@@ -0,0 +1,137 @@
+name: 'Automated tests'
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
Given that we use bbb-install-2.5.sh, it makes sense to reduce to `v2.5.x-release` and [for the time being] `develop` branches

Reply:
Indeed! Running in < 2.5 branch could report false positives. 

So you're proposing to target this PR to `v2.5.x-release` instead of develop?

Reply:
'develop' is fine as a pull request destination.
But the lines 

```
on:
  push:
    branches:
      - '*'
```

seem to imply that the action (when this PR is merged) will trigger on PRs to **any** BBB branch. I'd suggest it to only trigger on PRs to branch 'develop' (and maybe v2.5.x-release)

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags